js/src/dht/findprovs.js

```diff
@@ -60,7 +60,7 @@ module.exports = (createCommon, options) => {
         },
         (cidV0, cb) => nodeA.dht.findprovs(cidV0, cb),
         (provs, cb) => {
-          expect(provs.map((p) => p.id.toB58String()))
+          expect(provs.Responses.map((p) => p.ID))
```

Comment:
Can we please fix the object that is returned here to not have go-style uppercased first letter properties? - for consistency with the rest of the API.

Reply:
Documentation or implementation for `dht.findprovs` needs to be updated - `provs` is not an array of `PeerInfo` instances.

https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/DHT.md#dhtfindprovs

Reply:
updated